scripts/crontab

```diff
@@ -96,7 +93,7 @@ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/us
 15 12 * * * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash bounty_requests_email >> /var/log/gitcoin/bounty_requests_email.log  2>&1
 15 * * * * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash pending_start_work_actions  >> /var/log/gitcoin/pending_start_work_actions.log  2>&1
 #0 0 1 */3 * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash send_quarterly_stats --live  >> /var/log/gitcoin/send_quarterly_stats.log  2>&1
-00 10 * * 5 cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash post_leaderboard_to_slack  >> /var/log/gitcoin/post_leaderboard_to_slack.log  2>&1
+#00 10 * * 5 cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash post_leaderboard_to_slack  >> /var/log/gitcoin/post_leaderboard_to_slack.log  2>&1
```

Comment:
I think we can just completely retire this, maybe we want to add a new task to post it to discord?

Reply:
yes I I believe will be a future task to integrate discord into the system, when mattermost this task was changed to work with it instead of slack and the name wasn't changed. As it have all the query logic for leaderboard posts I left it at it is to help on the next integration.